style.css

```diff
@@ -0,0 +1,211 @@
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Museo Sans Cyrl 500';
+  font-style: normal;
+  font-size: 16px;
+  line-height: 19px;
+  color: #F8EEE4;
+  background: url(img/iswanto-arif-448003-unsplash.svg);
+  background-size: cover;
+
+}
+
+
+.wrapper {
+  margin: 0;
+}
+
+.header__wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-end;
+  justify-content: space-between;
+  padding-top: 145px;
+  margin-bottom: 265px;
+}
+
+.header__logo {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 247px;
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)